src/posts/2020-08-20-an-indepth-guide-to-aria-roles.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: An in-depth guide to ARIA roles
+description: ARIA roles are one half of the predefined categories of attributes used to describe elements that may not exist natively in browsers or may not be understood by screen readers and other assistive technologies.
+category: Background
+author: Duncan Jimbo
+date: 2020-08-20
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+
+The ARIA specification is divided into categories of attributes, one of which is Roles, and the other being States & Properties. ARIA roles, which are covered in this article, are generally used to describe elements that may not exist within HTML, or elements which may not have full cross-browser support, or have implementation gaps in screen readers and other assistive technologies. For instance, ARIA roles can be useful in situations where native HTML semantics are not understood in legacy user agents (e.g., browsers).
```

Comment:
>ARIA roles, which are covered in this article, are generally used to describe elements that may not exist within HTML, or elements which may not have full cross-browser support, or have implementation gaps in screen readers and other assistive technologies. 

This is a rather long sentence. What do you think about breaking it up so it reads: 

>ARIA roles, which are covered in this article, are generally used to describe elements: 
>
>- that may not exist within HTML, 
>- which may not have full cross-browser support, 
>- or have implementation gaps in screen readers and other assistive technologies.